ee/server/service/teams.go

```diff
@@ -836,7 +836,8 @@ func (svc *Service) createTeamFromSpec(
 		return nil, err
 	}
 
-	if enableDiskEncryption && defaults.MDM.EnabledAndConfigured {
+	if enableDiskEncryption && appCfg.MDM.EnabledAndConfigured {
+		// TODO: Are we missing an activity or anything else for BitLocker here?
```

Comment:
Looks like this is macOS (filevault)-specific code here? I.e. `appCfg.MDM.EnabledAndConfigured` is checking only for mac MDM and the calls in the `if` are apple-specific? (but maybe that's your point that an equivalent bitlocker logic is missing here?) EDIT: yeah I think that's what you meant, I agree with you it looks like something's missing.